options/posix/generic/sys-resource-stubs.cpp

```diff
@@ -3,15 +3,17 @@
 #include <sys/resource.h>
 
 #include <bits/ensure.h>
+#include <mlibc/debug.hpp>
 #include <mlibc/posix-sysdeps.hpp>
 
 int getpriority(int, id_t) {
 	__ensure(!"Not implemented");
 	__builtin_unreachable();
 }
+
 int setpriority(int, id_t, int) {
-	__ensure(!"Not implemented");
-	__builtin_unreachable();
+	mlibc::infoLogger() << "mlibc: setpriority is a stub!" << frg::endlog;
+	return 0;
 }
```

Comment:
It'd be better to stub this on the sysdep level. Or does Linux implement this on top of a different syscall?

Reply:
Stubbed with a `sys_setpriority` syscall.